tools/pot/docs/AccuracyAwareQuantizationUsage.md

```diff
@@ -0,0 +1,182 @@
+# Quantizing Model with Accuracy Control{#pot_accuracyaware_usage}
+
+@sphinxdirective
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   AccuracyAwareQuantization Method <accuracy_aware_README>
+
+@endsphinxdirective
+
+## Introduction
+This document assumes that you already tried [Default Quantization](@ref pot_default_quantization_usage) for the same model. In case when it introduces a significant accuracy degradation, Accuracy-aware Quantization algorithm can be used to remain accuracy within the pre-defined range. This may cause a 
```

Comment:
```suggestion
This document assumes that you already tried [Default Quantization](@ref pot_default_quantization_usage) for the same model. In case when it introduces a significant accuracy degradation, the Accuracy-aware Quantization algorithm can be used to remain accuracy within the pre-defined range. This may cause a 
```